experimenter/experimenter/experiments/models.py

```diff
@@ -1347,6 +1353,124 @@ def get_branch_data(self, analysis_basis, selected_segment):
 
         return branch_data
 
+    def get_metric_area_data(
+        self, metrics, analysis_basis, segment, reference_branch, window="overall"
+    ):
+        metric_data = {}
+        metric_area_data = {"metrics": metrics, "data": metric_data}
+
+        overall_results = (
+            (
+                self.results_data.get("v3", {})
+                .get("overall", {})
+                .get(analysis_basis, {})
+                .get(segment, {})
+            )
+            if self.results_data
+            else {}
+        )
+
+        for branch in self.get_sorted_branches():
+            branch_results = (
+                overall_results.get(branch.slug, {}).get("branch_data", {})
+                if isinstance(overall_results, dict)
+                else {}
+            )
+            branch_metrics = {}
+
+            for metric in metrics:
+                slug = metric.get("slug")
+                group = metric.get("group")
+
+                metric_src = branch_results.get(group, {}).get(slug, {})
+
+                absolute_data_list = metric_src.get("absolute", {}).get("all", [])
+                relative_data_list = (
+                    metric_src.get("relative_uplift", {})
+                    .get(reference_branch, {})
+                    .get("all", [])
+                )
+
+                def formattedAnalysisPointComparator(point):
+                    wi = point.get("window_index")
+                    return int(wi) if wi is not None else 0
+
+                absolute_data_list.sort(key=formattedAnalysisPointComparator)
+                relative_data_list.sort(key=formattedAnalysisPointComparator)
+
+                significance_map = (
+                    metric_src.get("significance", {})
+                    .get(reference_branch, {})
+                    .get(window, {})
+                )
+
+                abs_entries = []
+                for i, data_point in enumerate(absolute_data_list):
+                    lower = round(data_point.get("lower"), 2)
+                    upper = round(data_point.get("upper"), 2)
```

Comment:
We got a lot of feedback about showing more decimal points, so we added the ability to see more than 2 decimals on hover (we show 6). Can we keep that ability?

Reply:
Alright sounds good, as long as we have it on the todo list then no need to worry about it immediately.